programs/move_loader_api/src/lib.rs

```diff
@@ -59,9 +59,9 @@ pub enum LibraAccountState {
 #[derive(Default, Debug, Serialize, Deserialize)]
 pub struct InvokeInfo {
```

Comment:
Really don't like this struct.  `Vec<TransactionArgument>` should be sufficient.

Reply:
@jackcmay

Reply:
Looking into it